stl/inc/xutility

```diff
@@ -2074,32 +2074,42 @@ _NODISCARD _Ty _Fake_decay_copy(_Ty) noexcept;
 // (3) is non-throwing if and only if both conversion from decltype((E)) to T and destruction of T are non-throwing.
 
 namespace ranges {
+    // VARIABLE TEMPLATE _Has_complete_elements
+    template <class>
+    static constexpr bool _Has_complete_elements = false;
+
+    // clang-format off
+    template <class _Ty>
+        requires requires(_Ty& __t) { sizeof(__t[0]); }
+    static constexpr bool _Has_complete_elements<_Ty> = true;

```

Comment:
These are `static constexpr` at namespace scope. Did you intend `inline constexpr`?

Reply:
Ugh. yes.